getting_started/step_by_step/your_first_game.rst

```diff
@@ -1139,6 +1139,7 @@ functions:
 
     func _on_StartButton_pressed():
         $StartButton.hide()
+        $Message.hide()
         emit_signal("start_game")
 
     func _on_MessageTimer_timeout():
```

Comment:
The message should be hidden by the message timer below.

Reply:
Hmm, I can't check it right now, but it seems I forgot to connect the timeout signal.
Sorry for the troubles.